### PR TITLE
Convert toolchest to use same packaging as our other projects.

### PR DESCRIPTION
**Pull request by jguiditta**
Signed-off-by: Jason Guiditta <jguiditt@redhat.com>

### CODE REVIEW

**Comment by jguiditta** [comment]
Generated docs - https://test-toolchest.readthedocs.io/en/latest/readme.html

**Review by fuzzball81 — DISMISSED** [review]

File: README.rst [hunk 1]
```diff
@@ -16,41 +16,71 @@ toolchest
...
+To build the documentation on your checkout, simply run::
 
-https://toolchest.readthedocs.io.
+  make docs
 
 Contributing
-------------
+*************
+
+All new code should include tests that excercise the code and prove that it
```
Comment (jpichon) [inline README.rst:70]: (Nit) "exercise"

File: setup.cfg [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
+description = toolchest is a collection of generic functions for release-depot
+long_description = file: README.rst, HISTORY.rst
+long_description_content_type = text/x-rst
+url = https://github.com/release-depot/toolchest
+project_urls =
+    Bug Tracker = https://github.com/release-depot/toolchest/issues
+license = MIT
+license_files = LICENSE
+classifiers =
+    Development Status :: 4 - Best
```
Comment (jpichon) [inline setup.cfg:16]: Looks like "4" should be Beta, not Best? https://pypi.org/classifiers/

File: tox.ini [hunk 3]
```diff
@@ -5,18 +5,21 @@ envlist =
...
 
 
 [testenv:flake8]
 passenv=HOME
 sitepackages = False
-deps = flake8
 commands =
     flake8 --ignore=E501,W504,E741 setup.py docs toolchest tests
+
+[testenv:twine]
```
Comment (jpichon) [inline tox.ini:22]: Wonder if we should add a CI job using this, too? Not necessarily with this PR.

**Review by jpichon — COMMENTED** [review]
LGTM, just a couple of nits inline. I also tweaked the project settings so github stops looking for travis results.

File: setup.cfg (hunk 2 above)
Comment (jguiditta) [reply setup.cfg:16]: Ha, that is a hilarious typo, I'll fix these and update.

File: tox.ini (hunk 3 above)
Comment (jguiditta) [reply tox.ini:22]: Agree on adding it to CI. I don't mind adding it now, but I think if I leave it for a separate PR, we might be able to use this one to make sure mergify is now working correctly.

File: setup.cfg (hunk 2 above)
Comment (jpichon) [reply setup.cfg]: I thought it might be a typo for "Best effort" at first and went to check :-)

File: tox.ini (hunk 3 above)
Comment (jpichon) [reply tox.ini:22]: Makes sense to me!

**Review by jpichon — APPROVED** [review]

**Review by fuzzball81 — APPROVED** [review]

**Pull request merged**